services/api/tests/routes/test_endpoint.py

```diff
@@ -33,7 +33,7 @@ def test_endpoints_definition() -> None:
     splits = definition["/splits"]
     assert splits is not None
     assert splits["dataset"] is not None
-    assert len(splits["dataset"]) == 1  # Only has one processing step
+    assert len(splits["dataset"]) == 2  # Has two processing step
```

Comment:
Nit:
```suggestion
    assert len(splits["dataset"]) == 2  # Has two processing steps
```

Reply:
Thanks! I changed it to     `assert len(splits["dataset"]) == 3  # Has three processing steps` now that streaming worker is merged.